harwest/lib/utils/repository.py

```diff
@@ -26,9 +26,8 @@ def init(self):
         str(config.RESOURCES_DIR.joinpath("readme.template")),
         self.readme_path)
       git.add("README.md")
-      Initial_Commit = config.load_setup_data()['initcommitdate']
       git.commit(message="Initial commit with README.md",
-                 date=f"{Initial_Commit}", author=self.author)
+                 date=f"{str(datetime.now().strftime('%b/%d/%Y %H:%M'))}", author=self.author)
```

Comment:
Great stuff @s-i-d-d-i-s! Just a few minor things though:
- The `str(...)` over `strftime(...)` seems a bit redundant since it already has a return type of String: https://docs.python.org/3/library/datetime.html#datetime.date.strftime
- The f-string format in Python was introduced after version >= 3.6 https://www.python.org/dev/peps/pep-0498/ which might break our backward compatibility promise for Python 3.5 users https://github.com/nileshsah/harwest-tool/blob/master/setup.py#L37 I know 3.5 has already reached EOL but I fear that some people still might be using it. Maybe we'll end support for it too in a future release.

As for now, I wonder if a direct assignment `date=datetime.now().strftime('%b/%d/%Y %H:%M')` would simply work?

Reply:
> I wonder if a direct assignment `date=datetime.now().strftime('%b/%d/%Y %H:%M')` would simply work?

yeah it would work, i never took backward compatibility into consideration, but if that's a concern, then maybe we can avoid f-string? but i doubt anyone is still using python 3.5 or below

Reply:
I'd sure do agree with you that hardly anyone would be using Python 3.5 or below in this era, though you always find some 0.01% of the people who don't use Python much and are still happily on 3.5 or maybe even below. Hard to say without much stats but depreciating versions is one difficult job.
As for us, I think we can still breathe a bit more without using some of the fancy Python 3.6+ features. Don't think will hurt us much? I hope I haven't used them at any other place 😅 Do flag it if you see one!

Reply:
`Do flag it if you see one!`

haha sure